csharp/src/Ice/ConnectionI.cs

```diff
@@ -2556,7 +2556,7 @@ private void sendResponse(OutgoingResponse response, bool isTwoWay, byte compres
                         // Resume reading if the connection is active and the dispatch count is about to be less than
                         // _maxDispatches.
                         _threadPool.update(this, SocketOperation.None, SocketOperation.Read);
-                        _idleTimeoutTransceiver?.enableIdleCheck();
+                        _idleTimeoutTransceiver!.enableIdleCheck();
```

Comment:
Why can't be null here?

Reply:
It can be null - it's a bug. Reverted update.